package.json

```diff
@@ -10,11 +10,11 @@
     "csvexport": "^1.0.2",
     "date-fns": "^1.21.0",
     "director": "^1.2.8",
+    "gocommerce-js": "^1.1.1",
```

Comment:
I switched to this package since it looks like netlify-commerce-js was renamed to this.  Also looks like this package needs a release.